diode-server/cmd/ingester/main.go

```diff
@@ -5,18 +5,56 @@ import (
 	"os"
 
 	"github.com/getsentry/sentry-go"
+	"github.com/kelseyhightower/envconfig"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/metric"
 
 	"github.com/netboxlabs/diode/diode-server/ingester"
 	"github.com/netboxlabs/diode/diode-server/server"
+	"github.com/netboxlabs/diode/diode-server/telemetry"
+)
+
+const (
+	applicationName = "diode-ingester"
+	metricStartup   = "diode-ingester.startup_count"
```

Comment:
Should we standardise these with underscores instead of dashes, e.g.

```suggestion
	metricStartup   = "diode_ingester.startup_count"
```

Reply:
changed to `/` style... here's an example of how they render out in the prometheus exporter ```netboxlabs_com_diode_ingester_ingest_entity_count_total{hostname="f289f55008f5",otel_scope_name="netboxlabs.com/diode/ingester",otel_scope_version="",producer_app_name="diode-exampledata",producer_app_version="0.0.1",sdk_name="diode-sdk-python",sdk_version="0.0.0",stream="latest"} 1
netboxlabs_com_diode_ingester_ingest_request_count_total{hostname="f289f55008f5",otel_scope_name="netboxlabs.com/diode/ingester",otel_scope_version="",producer_app_name="diode-exampledata",producer_app_version="0.0.1",sdk_name="diode-sdk-python",sdk_version="0.0.0",stream="latest",success="true"} 1
netboxlabs_com_diode_ingester_startup_count_total{otel_scope_name="netboxlabs.com/diode/ingester",otel_scope_version=""} 1
netboxlabs_com_diode_reconciler_ingestion_processor_change_set_create_count_total{hostname="4730af2a1478",otel_scope_name="netboxlabs.com/diode/reconciler/ingestion_processor",otel_scope_version="",producer_app_name="diode-exampledata",producer_app_version="0.0.1",sdk_name="diode-sdk-python",sdk_version="0.0.0",success="false"} 1
netboxlabs_com_diode_reconciler_ingestion_processor_handle_message_count_total{hostname="4730af2a1478",otel_scope_name="netboxlabs.com/diode/reconciler/ingestion_processor",otel_scope_version="",producer_app_name="diode-exampledata",producer_app_version="0.0.1",sdk_name="diode-sdk-python",sdk_version="0.0.0",success="true"} 1
netboxlabs_com_diode_reconciler_ingestion_processor_ingestion_log_create_count_total{hostname="4730af2a1478",otel_scope_name="netboxlabs.com/diode/reconciler/ingestion_processor",otel_scope_version="",producer_app_name="diode-exampledata",producer_app_version="0.0.1",sdk_name="diode-sdk-python",sdk_version="0.0.0",success="true"} 1
netboxlabs_com_diode_reconciler_startup_count_total{otel_scope_name="netboxlabs.com/diode/reconciler",otel_scope_version=""} 1```

Reply:
The application name with '-' I left as-is since it is used for sentry and stopped referencing it for telemetry.